### PR TITLE
MRG, DOC: Add ECD to glossary

### PR DESCRIPTION
**Pull request by cjayb**
I noticed in conjunction with #6286 that ECD (or 'dipole' for that matter) wasn't defined in glossary. First stab here.

CC: @drammock

### CODE REVIEW

**Comment by cjayb** [comment]
I made the edits first, but willing to shorten & simplify!

An equivalent current dipole (ECD) is an ~idealized~``approximate`` representation of
~neuronal~``post-synaptic`` activity in a ~particular~ ``circumscribed/small`` region of cortex. The intracellular
currents that give rise to measurable EEG/MEG signals are thought to
originate in ~bundles~ ``populations?`` of cortical pyramidal neurons aligned
perpendicularly to the cortical surface. Because the length of ~those~``such``
current sources is very small relative to the distance between the
cortex and the EEG/MEG sensors, ~cortical current sources can be
well-modeled by~
the fields measured by the techniques are well-approximated
(*i.e.*, equivalent to) those generated by 
idealized point sources (dipoles) located on the
cortical surface.


```

File: doc/glossary.rst [hunk 1]
```diff
@@ -46,6 +46,20 @@ MNE-Python core terminology and general concepts
...
+
+    equivalent current dipole
+        An equivalent current dipole (ECD) is an approximate representation of
+        post-synaptic activity in a small region of cortex. The intracellular
+        currents that give rise to measurable EEG/MEG signals are thought to
+        originate in populations of cortical pyramidal neurons aligned
+        perpendicularly to the cortical surface. Because the length of such
+        current sources is very small relative to the distance between the
+        cortex and the EEG/MEG sensors, the fields measured by the techniques
+        are well-approximated by, *i.e.*, equivalent to those generated by
```
Comment (drammock) [inline doc/glossary.rst]: confused by the `*i.e.*` here. It took me a few reads of this sentence for it to make sense. How about this:

```suggestion
        are well-approximated by (i.e., "equivalent" to) fields generated by 
```

File: doc/glossary.rst [hunk 2]
```diff
@@ -46,6 +46,20 @@ MNE-Python core terminology and general concepts
         :func:`mne.bem.make_watershed_bem` or :func:`mne.bem.make_flash_bem`.
         See :ref:`tut-forward` for usage demo.
 
+    dipole
+        See 'equivalent current dipole'.
```
Comment (drammock) [inline doc/glossary.rst]: to be really thorough, this should be
```suggestion
        See :term:`equivalent current dipole`.
```
I realize they're adjacent, but in theory new terms could be added between them.

Comment (cjayb) [reply doc/glossary.rst]: that's just because I don't speak rst ;)

**Comment by drammock** [comment]
thanks @cjayb!

**Pull request merged**